src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperContentExtensions.cs

```diff
@@ -51,7 +52,18 @@ public static class TagHelperContentExtensions
 
```

Comment:
:ok: I guess. I'll see if de-weird-ing `LinkTagHelper` and `ScriptTagHelper` fits into one of my assigned "don't encode 'til absolutely necessary" bugs.